lxd/db/storage_volumes.go

```
@@ -106,7 +106,7 @@ SELECT DISTINCT node_id
 	for _, nodeID := range nodeIDs {
 		nodeVolumes, err := c.storagePoolVolumesGet(project, poolID, int64(nodeID), volumeTypes)
 		if err != nil {
-			return nil, err
+			continue
```

Maybe have it only skip if no rows are found so we don't hide all errors?